graphics/Circle.java

```diff
@@ -0,0 +1,79 @@
+package graphics;
+
+class Circle extends ShapeDesign{
+	
+    private int radious;
+    private final String SHAPE = "CIRCLE";
+    
+    public Circle(Point p,int radious){
+        this.p = p;
+        this.radious = radious;
+    }
+    
+    
+    
+    /**** Getter start********/
+    
+    public int getRadious(){
+    	return this.radious;
+    }
+    /*
+     *@return TYPE OF THIS SHAPE
+     */
+    public String getType(){
+    	return this.SHAPE;
+    }
+    
+    /******* Getter end***********/
+    
+    
+    
+    
+    
+    /**********ALL Public Method ********/
+    
+    
+    /*
+    *@return area of Circle(PI*r^2)
+    */
+    @Override
+    public double getArea(){
+        return (3.14*radious*radious);
+    }
+    
+    /*
+    *@return Perimeter of Circle (2*PI*r)
+    */
+    @Override
+    public double getPerimeter(){
+        return (2*3.14*radious);
```

Comment:
```suggestion
        return (2*Math.PI*radious);
```
We shall use java constants instead of writing values.